javascript/lifecycle.js

```diff
@@ -125,17 +126,29 @@ document.addEventListener(
 // - element - the element that triggered the reflex (not necessarily the Stimulus controller's element)
 //
 export const dispatchLifecycleEvent = (stage, element, reflexId) => {
+  const { reflexData, reflexController } = element
+
+  if (!document.body.contains(element)) {
```

Comment:
TIL `contains`! Never thought of running this at `document.body` scope. Slick man.